### PR TITLE
[MINOR] Fix license check exception

### PR DESCRIPTION
**Pull request by xwm1992**
Fix license check exception

### CODE REVIEW

File: tools/dependency-check/known-dependencies.txt [hunk 1]
```diff
@@ -28,18 +32,25 @@ grpc-protobuf-lite-1.17.1.jar
 grpc-stub-1.17.1.jar
 grpc-grpclb-1.17.1.jar
 gson-2.7.jar
+gson-2.8.2.jar
```
Comment (ruanwenjun) [inline tools/dependency-check/known-dependencies.txt:35]: Please unify the version, we can bound the version to 2.8.2.

File: tools/dependency-check/known-dependencies.txt [hunk 2]
```diff
@@ -28,18 +32,25 @@ grpc-protobuf-lite-1.17.1.jar
...
 httpclient-4.5.13.jar
 httpcore-4.4.13.jar
+httpcore-nio-4.4.6.jar
 ipaddress-5.3.3.jar
 j2objc-annotations-1.3.jar
 jackson-annotations-2.13.0.jar
 jackson-core-2.13.0.jar
 jackson-databind-2.13.0.jar
 javassist-3.20.0-GA.jar
+javassist-3.21.0-GA.jar
```
Comment (ruanwenjun) [inline tools/dependency-check/known-dependencies.txt:48]: Please unify the version.

File: tools/dependency-check/known-dependencies.txt [hunk 3]
```diff
@@ -118,23 +132,10 @@ simpleclient-0.8.1.jar
 simpleclient_common-0.8.1.jar
 simpleclient_httpserver-0.8.1.jar
 slf4j-api-1.7.30.jar
+snakeyaml-1.23.jar
+snakeyaml-1.30.jar
```
Comment (ruanwenjun) [inline tools/dependency-check/known-dependencies.txt:136]: Please unify the version.

**Review by qqeasonchen — APPROVED** [review]
LGTM

**Pull request merged**